doc/tutorials/SIR-X.ipynb

```diff
@@ -387,7 +355,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4-final"
+   "version": "3.8.1"
```

Comment:
Was this an intended change?

Reply:
This was change automatically by Jupyter when opened from my computer instead of Pipenv. In practice this unavoidable unless we use the development setup here as a reference.

What do you think?

Reply:
I used the pipenv setup and it points now to 3.7.7. I think we don't have control of this.

Reply:
Ah it's no problem then. I do not know much about Jupyter, that's why I was asking haha